app/services/discovery_report.rb

```diff
@@ -23,11 +23,6 @@ def output_path
     Dir::Tmpname.create(["#{self.class.name.underscore}_", '.json'], batch.batch_context.output_dir) { |path| path }
   end
 
-  # @return [Boolean]
-  def using_file_manifest?
-    file_manifest&.exists?
-  end
-
```

Comment:
**refactor:** removed this unused method which was using that janky `exists?` method on the file manifest object